src/pages/postgraphile/required-knowledge.md

```diff
@@ -176,3 +176,45 @@ create policy friend_insert on my_other_table for insert with check ( are_friend
   that action (doesn't make sense for `SELECT` or `DELETE`)
 - NOTE: if `with check` is not specified then the `using` clause will also be
   used for `with check` automatically.
+
+#### Recommended path
+
+A good way to tackle and make a sense of postgraphile is to go through these
+steps:
```

Comment:
What do you think about adding this talk here? https://www.youtube.com/watch?v=BNLcHlMn5X4

Reply:
Either that or add it to the list in the README in the root directory `./README.md`
As I understand it gives the intuition as to why it is faster to do DDD.

Reply:
@jemgillam do you have opinions on this?

Reply:
The talk would fit better in the README I think, as it is a more general introduction to this way of thinking